src/Service/RedirectedURLService.php

```diff
@@ -0,0 +1,133 @@
+<?php
+
+namespace SilverStripe\RedirectedURLs\Service;
+
+use SilverStripe\Control\Director;
+use SilverStripe\Control\HTTPRequest;
+use SilverStripe\Control\HTTPResponse;
+use SilverStripe\Control\HTTPResponse_Exception;
+use SilverStripe\Core\Config\Configurable;
+use SilverStripe\Core\Convert;
+use SilverStripe\Core\Extensible;
+use SilverStripe\ORM\ArrayList;
+use SilverStripe\RedirectedURLs\Model\RedirectedURL;
+
+class RedirectedURLService
```

Comment:
Maybe define an interface for this service and add some documentation for the class.